build/Dockerfile.ci

```diff
@@ -78,8 +78,7 @@ RUN env -u VERSION GOOS=windows make ecr-credential-provider
 # Build CNI plugins
 WORKDIR /build/windows-machine-config-operator/containernetworking-plugins/
 COPY containernetworking-plugins/ .
-ENV CGO_ENABLED=0
-RUN ./build_windows.sh
+RUN CGO_ENABLED=1 CC=x86_64-w64-mingw32-gcc ./build_windows.sh
```

Comment:
do we need this change in the midstream dockerfiles as well?

Reply:
yes we will

Reply:
unless we know that 4.18 will be released with konflux

Reply:
> unless we know that 4.18 will be released with konflux
That is the plan but please do it anyway so QE is able to test before the transition occurs